.github/workflows/ci.yml

```
@@ -37,12 +37,12 @@ jobs:
       -
         name: Set tag for release
         if: ${{ matrix.version == 'release' }}
-        run: printf 'TAG=' && curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' >> $GITHUB_ENV
+        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/TAG=\1/' >> $GITHUB_ENV
```

```suggestion
        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's|.*"([^"]+)".*|TAG=docker.io/mpepping/cyberchef:\1|' >> $GITHUB_ENV
```
Sorry, this broke with my last PR

---

No worries! It's an opportunity to improve on our GitHub Actions skills right. :)